main.py

```diff
@@ -1,7 +1,37 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Actor, Genre
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Drama"]
+
+    for item in genres:
+        Genre.objects.create(name=item)
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for first_name, last_name in actors:
+        Actor.objects.create(
+            first_name=first_name,
+            last_name=last_name
+        )
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
```

Comment:
It's working in this case, but it's better to add the first name to the filter according to the task 

![image](https://user-images.githubusercontent.com/104218550/224710101-f0e3759f-58b8-4dcd-96df-e28a54daf0e1.png)